### ISSUE TITLE
[V5] Downloaded images not available anymore without running Gatsby Clean first

### ISSUE THREAD

**Issue by DennisKraaijeveld** [prismicio/prismic-gatsby#471]
### Versions
-"gatsby": "^4.1.0",
-"gatsby-plugin-sharp": "^4.1.0",
-"gatsby-source-prismic": "^5.0.2",
- node: v16.12.0

### Reproduction

Will make a reproduction tomorrow.

### Steps to reproduce
I did set shouldDownloadFiles to true for a specific file like this:

```
 shouldDownloadFiles: {
          "homepage.data.body.text_section.primary.image_one": true,
        },
```

Before upgrading from `5.0.0` to `5.0.2` everything was working like expected. But now the image will only be available on localFile on first build after a `gatsby clean`. When running `gatsby develop` without running a `gatsby clean` first, the image will not be available.

**Comment by angeloashmore** [comment]
Thanks @DennisKraaijeveld! I just published `v5.0.3` with a fix that should that issue.

It turned out to be a garbage collection issue. More details are given here: https://github.com/prismicio/prismic-gatsby/pull/472

**Comment by DennisKraaijeveld** [comment]
Graci!
